x-pack/plugins/maps/public/lens/choropleth_chart/choropleth_chart.tsx

```diff
@@ -126,7 +128,11 @@ export function ChoroplethChart({
     type: LAYER_TYPE.GEOJSON_VECTOR,
   };
 
-  return <PassiveMap passiveLayer={choroplethLayer} onRenderComplete={onRenderComplete} />;
+  return (
```

Comment:
Should it make more sense to move KibanaRednerContextProvider to [x-pack/plugins/maps/public/lens/choropleth_chart/expression_renderer.tsx](https://github.com/elastic/kibana/blob/main/x-pack/plugins/maps/public/lens/choropleth_chart/expression_renderer.tsx#L100) since that is where the react tree starts?

Reply:
b002b1090f8